modules/core/src/main/java/org/apache/ignite/internal/cluster/IgniteClusterEx.java

```diff
@@ -40,27 +34,4 @@
      * @return Unique cluster ID.
      */
     public UUID id();
```

Comment:
Cluster ID is part of the same functionality as Cluster Tag and should be republished back to IgniteCluster interface as well.

After that we should delete IgniteClusterEx interface as unnecessary.

Reply:
done